src/main/java/racingcar/Application.java

```diff
@@ -1,7 +1,24 @@
 package racingcar;
+import javax.lang.model.element.Name;
+import java.util.Scanner;
+import java.util.ArrayList;
+import static camp.nextstep.edu.missionutils.Console.readLine;
 
 public class Application {
     public static void main(String[] args) {
         // TODO: 프로그램 구현
+        ArrayList<Car> cars = new ArrayList<>();
+        System.out.println("경주할 자동차 이름을 입력하세요.");
+        String name = readLine();
+        String[] Name = name.split(",");
+        for(String n: Name){
```

Comment:
`n` 이라는 변수명보다는 `name`으로 사용하면 좋을것 같아요
```suggestion
        for(String name: names){
```